docs/source/framework/pytorch_integration/autograd_with_tc.rst

```diff
@@ -27,11 +27,11 @@ Examples
      from torch.nn.parameter import Parameter
      CONV_LANG = """
      def convolution(float(N,C,H,W) I, float(M,C,KH,KW) W1) -> (O) {{
-        O(n, m, h, w) +=! I(n, c, {sh} * h + kh, {sw} * w + kw) * W1(m, c, kh, kw)
+       O(n, m, h, w) +=! I(n, r_c, {sh} * h + r_kh, {sw} * w + r_kw) * W1(m, r_c, r_kh, r_kw)
      }}
-     def convolution_grad(float(N,C,H,W) I, float(M,C,KH,KW) W1, float(N,M,H,W) O_grad) -> (I_grad, W1_grad) {{
-        I_grad(n, c, h, w) +=! O_grad(n, m, {sh} * h - kh, {sw} * w - kw) * W1(m, c, kh, kw)
-        W1_grad(m, c, kh, kw) +=! O_grad(n, m, {sh} * h - kh, {sw} * w - kw) * I(n, c, h, w)
+     def convolution_grad(float(N,C,H,W) I, float(M,C,KH,KW) W1, float(N,M,H,W) g_O) -> (g_I, g_W1) {{
+        g_I(n, c,  h,  w) +=! g_O(  n, r_m, {sh} *   h - r_kh, {sw} *   w - r_kw) * W1(r_m, c, r_kh, r_kw)
```

Comment:
we never said that we prefer aligning

```
  X(i, j)
FOO(m,n)
```

over
```
X  (i,j)
FOO(i,j)
```

Reply:
we didn't, use your best judgement, do you see an issue with this alignment?